src/content/id/fundamentals/codelabs/your-first-pwapp/index.md

```diff
@@ -77,37 +77,37 @@ Untuk menguji apakah Kunci API Anda berfungsi dengan benar, buat permintaan HTTP
 
 ### Dapatkan kodenya
 
-Kami telah memasukkan semua yang Anda butuhkan untuk proyek ini ke dalam repo Git. Untuk memulai, Anda harus mengambil kode dan membukanya di lingkungan pengembang favorit Anda. Untuk codelab ini, kami sarankan menggunakan Glitch.
+Kita telah memasukkan semua yang Anda butuhkan untuk proyek ini ke dalam repo Git. Untuk memulai, Anda harus mengambil kode dan membukanya di lingkungan pengembang (IDE) favorit Anda. Untuk codelab ini, kami sarankan menggunakan Glitch.
 
 #### Sangat Dianjurkan: Gunakan Glitch untuk mengimpor repo
 
 Menggunakan Glitch adalah metode yang disarankan untuk bekerja melalui codelab ini.
 
-1. Buka tab browser baru dan pergi ke [https://glitch.com](https://glitch.com) .
+1. Buka tab browser baru dan pergi ke [https://glitch.com](https://glitch.com).
 2. Jika Anda tidak memiliki akun, Anda harus mendaftar.
 3. Klik __New Project__, lalu __Klik dari Git Repo .__
-4. Klon __https: //github.com/googlecodelabs/your-first-pwapp.git__ dan klik OK.
+4. Duplikasi (_clone_) __https: //github.com/googlecodelabs/your-first-pwapp.git__ dan klik OK.
 5. Setelah repo dimuat, edit file `.env` , dan perbarui dengan kunci API DarkSky Anda. 6. Klik tombol __Tampilkan Live__ untuk melihat PWA dalam aksi.
 
 #### Alternatif: Unduh kode &amp; bekerja secara lokal
 
 Jika Anda ingin mengunduh kode dan bekerja secara lokal, Anda harus memiliki Node versi terbaru, dan pengaturan editor kode dan siap untuk digunakan.
 
-Caution: Jika Anda bekerja secara lokal, beberapa audit Lighthouse tidak akan berlalu, dan instalasi mungkin tidak tersedia karena server lokal tidak menyajikan konten dalam konteks yang aman.
+Perhatian: Jika Anda bekerja secara lokal, beberapa audit Lighthouse tidak akan terpenuhi, dan instalasi mungkin tidak tersedia karena server lokal tidak menyajikan konten dalam konteks yang aman.
```

Comment:
Please leave lines starting with "Caution:" untranslated, it's a special keyword that does some extra formatting. The same goes for "Note:", "Warning:", "Success:", "Key Point:", "Key Term:", and "Objective:"

Would you update throughout the article please?

Reply:
Yes, I will